src/madness/world/thread.h

```diff
@@ -1291,9 +1285,9 @@ namespace madness {
 	}
 
 #if HAVE_PARSEC
-	////////////////// Parsec Related Begin //////////////////
-        static parsec_context_t *parsec;
-        ///////////////// Parsec Related End ////////////////////
+	    ////////////////// Parsec Related Begin //////////////////
+	    static ParsecRuntime *parsec_runtime;
```

Comment:
Why is parsec_runtime static? It's deleted in the dtor of Taskpool (and not set to `null`) so it should be a member of the taskpool, no?

Reply:
begin(), end(), and add(), that all call methods of this object, are all defined static, so this pulled all those to become static too.

The ParsecRuntime class holds a parsec_context_t and a parsec_taskpool_t, so I'm not sure I understand your question about the ParsecRuntime being a member of the taskpool.